dist/pathfora.js

```diff
@@ -2772,9 +2778,131 @@
     this.utils = utils;
   };
 
+  /**
+   * @class
+   * @name Inline
+   * @description Inline Personalization
+   */
+  Inline = function () {
+    this.lioElements = [];
```

Comment:
probably dont need namespacing here and if we do it should e related to pathfora and not lio since the two libraries are technically independent.